tests/wp-unit/include/Core/Metas/Modules/DeleteCommentMetaModuleTest.php

```diff
@@ -0,0 +1,588 @@
+						'meta_key'           => 'test_key',
+						'meta_value'         => '10',
+						'number_of_comments' => 5,
+					),
+					'miss_matched' => array(
+						'meta_key'           => 'test_key',
+						'meta_value'         => '20',
+						'number_of_comments' => 3,
+					),
+				),
+				array(
+					'meta_key'   => 'test_key',
+					'meta_type'  => 'NUMERIC',
+					'meta_value' => '10',
+					'operator'   => '=',
+				),
+				array(
+					'matched'      => '5',
+					'miss_matched' => '3',
+				),
+			),
+
+			array(
+				array(
+					'matched'      => array(
+						'meta_key'           => 'test_key',
+						'meta_value'         => '10',
+						'number_of_comments' => 5,
+					),
+					'miss_matched' => array(
+						'meta_key'           => 'test_key',
+						'meta_value'         => '20',
+						'number_of_comments' => 3,
+					),
+				),
+				array(
+					'meta_key'   => 'test_key',
+					'meta_type'  => 'NUMERIC',
+					'meta_value' => '10',
+					'operator'   => '!=',
+				),
+				array(
+					'matched'      => '3',
+					'miss_matched' => '0',
+				),
+			),
+
+		);
+	}
+
+	/**
+	 * Add to test deleting comment meta from more than one comment using meta value as well with different operations.
+	 *
+	 * @param array $input create posts, comments and meta params.
+	 * @param array $opetation Possible operations.
+	 * @param array $expected expected output.
+	 *
+	 * @dataProvider provide_data_to_test_that_comment_meta_from_multiple_comments
+	 */
+	public function test_that_comment_meta_from_multiple_comments( $input, $opetation, $expected ) {
+		$post_type = 'post';
+
+		// Create a post.
+		$post = $this->factory->post->create(
+			array(
+				'post_type' => $post_type,
+			)
+		);
+
+		$comment_data = array(
+			'comment_post_ID' => $post,
+		);
+
+		foreach ( $input as $meta ) {
+			for ( $i = 0; $i < $meta['number_of_comments']; $i++ ) {
+				$comment_id = $this->factory->comment->create( $comment_data );
+				add_comment_meta( $comment_id, $meta['meta_key'], $meta['meta_value'] );
+			}
+		}
+
+		$delete_options = array(
+			'post_type'  => $post_type,
+			'use_value'  => true,
+			'meta_key'   => $opetation['meta_key'],
+			'meta_value' => $opetation['meta_value'],
+			'meta_type'  => $opetation['meta_type'],
+			'meta_op'    => $opetation['operator'],
+			'limit_to'   => 0,
+			'date_op'    => '',
+			'days'       => '',
+			'restrict'   => false,
+		);
+
+		$comment_metas_deleted = $this->module->delete( $delete_options );
+		$this->assertEquals( $expected['matched'], $comment_metas_deleted );
+
+		$args = array(
+			'post_type'  => $post_type,
+			'meta_key'   => $input['miss_matched']['meta_key'],
+			'meta_value' => $input['miss_matched']['meta_value'],
+		);
+
+		$comments = get_comments( $args );
+
+		$this->assertEquals( $expected['miss_matched'], count( $comments ) );
+
+	}
+
+	/**
+	 * Add to test deleting comment meta older than x days.
+	 */
+	public function test_that_comment_meta_can_be_deleted_olderthan_x_days() {
```

Comment:
Change the method name to `test_that_comment_meta_posts_older_than_x_days_can_be_deleted`. Also check the method name format for other test casts as well.

Reply:
Fixed